src/common/components/link/Link.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link as HDSLink, LinkProps as HDSLinkProps } from "hds-react";
+import { useTranslation } from "next-i18next";
+
+import getIsHrefExternal from "../../utils/getIsHrefExternal";
+
+type Props = Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, "children"> & {
+  children: HDSLinkProps["children"];
+  disableVisitedStyles?: HDSLinkProps["disableVisitedStyles"];
+};
+
+const Link = React.forwardRef<HTMLAnchorElement, Props>(
+  ({ href, target, ...delegatedProps }, ref) => {
+    const { t } = useTranslation("link");
+
+    const isOpenInNewTab = target === "_blank";
+    const isExternal = getIsHrefExternal(href);
+
+    return (
+      // Apply ref directly to HDSLink once it supports it
+      <span ref={ref}>
+        <HDSLink
+          {...delegatedProps}
+          href={href}
+          openInNewTab={isOpenInNewTab}
+          external={isExternal}
+          size="M"
```

Comment:
should size be configurable as well with some default?

Reply:
I attempted to use this size property to adjust font sizes for some components. The issue is that in HDS the icon scales more than the text, which is in conflict with the design that has been made for this service.

<img width="258" alt="Screenshot 2022-04-13 at 7 59 41" src="https://user-images.githubusercontent.com/9090689/163103647-5427cf5d-411f-4e89-b686-7ef581b4edf5.png">

My thinking was that for now, using the prop would most likely lead to a result that conflicts with the design. And in that way it doesn't make sense to expose the prop. It should be safe to expose later on as well if the need arises.

Reply:
I've exposed the prop for now 👍